sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/decrypt_parameters.hpp

```diff
@@ -68,6 +63,24 @@ namespace Azure {
      */
     DecryptParameters() = delete;
 
+    /**
+     * @brief Gets the initialization vector for decryption.
+     *
+     */
+    std::vector<uint8_t> Iv;
```

Comment:
Do other languages also call this abbreviated as `Iv`? As a naiive user, I would have no idea what this means, and would be required to read docs.

I recommend we call it `InitializationVector` to be self-documenting unless all the other languages use this short form.

Reply:
Looks like other languages use this short form:
https://docs.microsoft.com/en-us/dotnet/api/azure.security.keyvault.keys.cryptography.decryptparameters.iv?view=azure-dotnet#Azure_Security_KeyVault_Keys_Cryptography_DecryptParameters_Iv
https://docs.microsoft.com/en-us/java/api/com.azure.security.keyvault.keys.cryptography.decryptparameters.createa256cbcpadparameters?view=azure-java-preview

@heaths, @tg-msft what do you think?

Reply:
We don't abbreviate aad, but rather state the whole thing `AdditionalAuthenticatedData` even though the REST API doc use `aad`:
https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt#request-body

 @heaths, @tg-msft would you stand by such a name if you had to redo this?
https://docs.microsoft.com/en-us/dotnet/api/azure.security.keyvault.keys.cryptography.encryptparameters.additionalauthenticateddata?view=azure-dotnet#Azure_Security_KeyVault_Keys_Cryptography_EncryptParameters_AdditionalAuthenticatedData

Reply:
"aad" is not an crypto industry term. "iv" is (initialization vector). So, yes, I'd still use "iv" because it's as ubiquitous in crypto as "id" is everywhere.

Reply:
Sounds good. Marking this thread as resolved since no action is needed.